cmp/dataview/DataView.ts

```diff
@@ -37,18 +38,11 @@ export const [DataView, dataView] = hoistCmp.withFactory({
     }
 });
 
-DataView.propTypes = {
-    /** Primary component model instance. */
-    model: PT.oneOfType([PT.instanceOf(DataViewModel), PT.object])
-};
-
 class DataViewLocalModel extends HoistModel {
     xhImpl = true;
 
-    /** @member {DataViewModel} */
-    @lookup(DataViewModel) model;
-
-    agOptions;
+    @lookup(DataViewModel) model: DataViewModel;
+    agOptions: PlainObject;
```

Comment:
Does AgGrid provide a type for this?

Reply:
Not typing any agGrid properties at this time.